dts/bindings/i2c/i2c.yaml

```diff
@@ -27,7 +27,12 @@ properties:
       type: int
       category: required
       description: should be 0.
-    clock-frequency :
+    input-frequency:
```

Comment:
AFAICT `input-frequency` is not a standard Linux device-tree property for I2C (it's not documented for any devicetree binding).  Assuming we're supposed to follow Linux and SiFive requires this information I believe this belongs in `sifive,i2c0.yaml` instead.

Though it doesn't appear to be used there, either.  So the commit message needs more justification on why Zephyr needs this.

Reply:
I'll move it to `sifive,i2c0.yaml`.

This is used, the driver uses it to set clock dividers to produce the correct bus frequency.